wiki/Tournaments/OWC/2018/en.md

```diff
@@ -1,21 +1,21 @@
 ---
```

Comment:
should rename Grand Finals -> Finals week 2 and Finals -> Finals week 1 
like in previous installments